### PR TITLE
fix(storybook): [ABI-5014] making Storybook works with EXPO SDK 48

### PR DESCRIPTION
**Pull request by elenitaex5**
# Description

Fixing all dependencies and files  making storybook works after upgrading Expo SDK 48

# Checklist:

- [X] Storybook has been updated (if needed) and still working
- [ ] Tests have been run locally before create the PR

---

- [ ] Tested in Android simulator
- [ ] Tested in iOS simulator
- [ ] Tested in Android device through Expo Go
- [ ] Tested in Android device as app
- [ ] Tested in iOS device through Expo Go
- [ ] Tested in iOS device as app
- [ ] Tested in web

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -8,11 +9,15 @@ export const parameters = {
...
 export const decorators = [
-  Story => (
-    <View style={{ height: 'inherit' }}>
+  (Story) => (
+    <View
+      style={{
+        flex: 1,
+        backgroundColor: '#ffffff',
+        padding: 8
+      }}
```
Comment (ylagef) [inline .storybook/preview.js:19]: This inline style and this color not used as constant can be improved

File: App.tsx [hunk 2]
```diff
@@ -57,4 +58,12 @@ export const App = () => {
   }
 }
 
-export default App
+let AppEntryPoint = App
+
+console.log(Constants?.expoConfig, Constants?.expoConfig?.extra?.storybookEnabled)
```
Comment (ylagef) [inline App.tsx:63]: Can this log be removed?
```suggestion
```

**Review by ylagef — APPROVED** [review]
Please check my suggestions before merging.

File: .storybook/preview.js (hunk 1 above)
Comment (elenitaex5) [reply .storybook/preview.js]: Solved in 335acbd commit

**Pull request merged**